portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase

```diff
@@ -39,12 +39,25 @@ definition {
 	}
 
 	@description = "Automation ID: LRAC-8107 | Test Summary: Navigates to data source page"
-	@ignore = "true"
 	@priority = "5"
 	test CanNavigatesToDataSourcePage {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		ACDXPSettings.syncNewContactData();
+
+		Navigator.openURL();
+
+		Pause(locator1 = "5000");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
 
-		// TODO LRAC-8107 CanNavigatesToDataSourcePage
+		ACSettings.assertDataSourceList(dataSourceName = "Beryl Commerce");
 
+		ACSettings.assertDataSourceList(dataSourceName = "Beryl Salesforce");
```

Comment:
Hey @calvinctvo , what do you think about also validate if the newly added Data Source is also available on the list?

Look for this implementation Josh did a few days ago to extract this info based on the value of the propertyName var.

https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase#L37

Reply:
He did: 

`var dataSource = StringUtil.replace("${propertyName}", " Combined Property", "");`

Then you can validate this info too, and not only the Beryl Commerce or Beryl Salesforce.